frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -696,7 +696,7 @@ export const topologyModelFromDataModel = (
         type: d.type,
         label: dataModel.topology[d.id].name,
         data,
-        collapsed: filters && !filters.display.knativeServices,
+        collapsed: d.type === TYPE_KNATIVE_SERVICE && filters && !filters.display.knativeServices,
```

Comment:
@jeff-phillips-18 We need to support `TYPE_OPERATOR_BACKED_SERVICE` as well

Reply:
Yes, that is story https://issues.redhat.com/browse/ODC-2851, not part of this PR.